File: AutomaticHeightTagTableViewCell/Models/TagGroups.m
//
//  TagGroups.m
//  AutomaticHeightTagTableViewCell
//
//  Created by WEI-JEN TU on 2016-07-16.
//  Copyright © 2016 Cold Yam. All rights reserved.
//

#import "TagGroups.h"

@implementation TagGroups

+ (NSArray<NSArray<AHTag *> *> *)dataSource {
    NSArray *pinterest = @[[self pin1], [self pin2], [self pin3], [self pin4]];
    NSArray *genre = @[[self genre1], [self genre2], [self genre3], [self genre4], [self genre5], [self genre6], [self genre7], [self genre8], [self genre9], [self genre10]];
    NSArray *device = @[[self device1], [self device2], [self device3], [self device4], [self device5], [self device6], [self device7], [self device8], [self device9]];
    NSArray *app = @[[self app1], [self app2], [self app3], [self app4], [self app5], [self app6], [self app7], [self app8], [self app9], [self app10], [self app11], [self app12], [self app13], [self app14], [self app15], [self app16], [self app17], [self app18], [self app19], [self app20], [self app22], [self app22], [self app23], [self app24]];
    return @[pinterest, genre, device, app];
}

#pragma mark - Pinterest

+ (AHTag *)pin1 {
    AHTag *tag = [AHTag new];
    tag.category = @"Pinterest";
    tag.title = @"All Pins";
    tag.color = [UIColor colorWithRed:1 green:0.56 blue:0.56 alpha:1];
    tag.url = [NSURL URLWithString:@"http://pinterest.com"];
    tag.enabled = @YES;
    return tag;
}

+ (AHTag *)pin2 {
    AHTag *tag = [AHTag new];
    tag.category = @"Pinterest";
    tag.title = @"Your Pins";
    tag.color = [UIColor colorWithRed:1 green:0.56 blue:0.56 alpha:1];
    tag.url = [NSURL URLWithString:@"http://pinterest.com"];
    tag.enabled = @YES;
    return tag;
}

+ (AHTag *)pin3 {
    AHTag *tag = [AHTag new];
    tag.category = @"Pinterest";
    tag.title = @"People";
    tag.color = [UIColor colorWithRed:1 green:0.56 blue:0.56 alpha:1];
    tag.url = [NSURL URLWithString:@"http://pinterest.com"];
    tag.enabled = @NO;
    return tag;
}

+ (AHTag *)pin4 {
    AHTag *tag = [AHTag new];
    tag.category = @"Pinterest";
    tag.title = @"Boards";
    tag.color = [UIColor colorWithRed:1 green:0.56 blue:0.56 alpha:1];
    tag.url = [NSURL URLWithString:@"http://pinterest.com"];
    tag.enabled = @YES;
    return tag;
}

#pragma mark - Genere

+ (AHTag *)genre1 {
    AHTag *tag = [AHTag new];
    tag.category = @"Genre";
    tag.title = @"Classical";
    tag.color = [UIColor colorWithRed:0.626 green:0.56 blue:1 alpha:1];
    tag.url = [NSURL URLWithString:@"http://itunes.com"];
    tag.enabled = @YES;
    return tag;
}

+ (AHTag *)genre2 {
    AHTag *tag = [AHTag new];
    tag.category = @"Genre";
    tag.title = @"Comedy";
    tag.color = [UIColor colorWithRed:0.626 green:0.56 blue:1 alpha:1];
    tag.url = [NSURL URLWithString:@"http://itunes.com"];
    tag.enabled = @YES;
    return tag;
}

+ (AHTag *)genre3 {
    AHTag *tag = [AHTag new];
    tag.category = @"Genre";
    tag.title = @"Country";
    tag.color = [UIColor colorWithRed:0.626 green:0.56 blue:1 alpha:1];
    tag.url = [NSURL URLWithString:@"http://itunes.com"];
    tag.enabled = @YES;
    return tag;
}

+ (AHTag *)genre4 {
    AHTag *tag = [AHTag new];
    tag.category = @"Genre";
    tag.title = @"Dance";
    tag.color = [UIColor colorWithRed:0.626 green:0.56 blue:1 alpha:1];
    tag.url = [NSURL URLWithString:@"http://itunes.com"];
    tag.enabled = @NO;
    return tag;
}

+ (AHTag *)genre5 {
    AHTag *tag = [AHTag new];
    tag.category = @"Genre";
    tag.title = @"Drama";
    tag.color = [UIColor colorWithRed:0.626 green:0.56 blue:1 alpha:1];
    tag.url = [NSURL URLWithString:@"http://itunes.com"];
    tag.enabled = @YES;
    return tag;
}

+ (AHTag *)genre6 {
    AHTag *tag = [AHTag new];
    tag.category = @"Genre";
    tag.title = @"Early Listening";
    tag.color = [UIColor colorWithRed:0.626 green:0.56 blue:1 alpha:1];
    tag.url = [NSURL URLWithString:@"http://itunes.com"];
    tag.enabled = @YES;
    return tag;
}

+ (AHTag *)genre7 {
    AHTag *tag = [AHTag new];
    tag.category = @"Genre";
    tag.title = @"Electronic";
    tag.color = [UIColor colorWithRed:0.626 green:0.56 blue:1 alpha:1];
    tag.url = [NSURL URLWithString:@"http://itunes.com"];
    tag.enabled = @YES;
    return tag;
}

+ (AHTag *)genre8 {
    AHTag *tag = [AHTag new];
    tag.category = @"Genre";
    tag.title = @"Folk";
    tag.color = [UIColor colorWithRed:0.626 green:0.56 blue:1 alpha:1];
    tag.url = [NSURL URLWithString:@"http://itunes.com"];
    tag.enabled = @NO;
    return tag;
}

+ (AHTag *)genre9 {
    AHTag *tag = [AHTag new];
    tag.category = @"Genre";
    tag.title = @"Hip Hop/Rap";
    tag.color = [UIColor colorWithRed:0.626 green:0.56 blue:1 alpha:1];
    tag.url = [NSURL URLWithString:@"http://itunes.com"];
    tag.enabled = @YES;
    return tag;
}

+ (AHTag *)genre10 {
    AHTag *tag = [AHTag new];
    tag.category = @"Genre";
    tag.title = @"Holiday";
    tag.color = [UIColor colorWithRed:0.626 green:0.56 blue:1 alpha:1];
    tag.url = [NSURL URLWithString:@"http://itunes.com"];
    tag.enabled = @YES;
    return tag;
}

#pragma mark - Device

+ (AHTag *)device1 {
    AHTag *tag = [AHTag new];
    tag.category = @"Device";
    tag.title = @"Mobile";
    tag.color = [UIColor colorWithRed:0.56 green:0.713 blue:1 alpha:1];
    tag.url = [NSURL URLWithString:@"http://www.spotify.com"];
    tag.enabled = @YES;
    return tag;
}

+ (AHTag *)device2 {
    AHTag *tag = [AHTag new];
    tag.category = @"Device";
    tag.title = @"Computer";
    tag.color = [UIColor colorWithRed:0.56 green:0.713 blue:1 alpha:1];
    tag.url = [NSURL URLWithString:@"http://www.spotify.com"];
    tag.enabled = @YES;
    return tag;
}

+ (AHTag *)device3 {
    AHTag *tag = [AHTag new];
    tag.category = @"Device";
    tag.title = @"Tablet";
    tag.color = [UIColor colorWithRed:0.56 green:0.713 blue:1 alpha:1];
    tag.url = [NSURL URLWithString:@"http://www.spotify.com"];
    tag.enabled = @YES;
    return tag;
}

+ (AHTag *)device4 {
    AHTag *tag = [AHTag new];
    tag.category = @"Device";
    tag.title = @"Car";
    tag.color = [UIColor colorWithRed:0.56 green:0.713 blue:1 alpha:1];
    tag.url = [NSURL URLWithString:@"http://www.spotify.com"];
    tag.enabled = @NO;
    return tag;
}

+ (AHTag *)device5 {
    AHTag *tag = [AHTag new];
    tag.category = @"Device";
    tag.title = @"Speaker";
    tag.color = [UIColor colorWithRed:0.56 green:0.713 blue:1 alpha:1];
    tag.url = [NSURL URLWithString:@"http://www.spotify.com"];
    tag.enabled = @YES;
    return tag;
}

+ (AHTag *)device6 {
    AHTag *tag = [AHTag new];
    tag.category = @"Device";
    tag.title = @"Playstation";
    tag.color = [UIColor colorWithRed:0.56 green:0.713 blue:1 alpha:1];
    tag.url = [NSURL URLWithString:@"http://www.spotify.com"];
    tag.enabled = @YES;
    return tag;
}

+ (AHTag *)device7 {
    AHTag *tag = [AHTag new];
    tag.category = @"Device";
    tag.title = @"TV";
    tag.color = [UIColor colorWithRed:0.56 green:0.713 blue:1 alpha:1];
    tag.url = [NSURL URLWithString:@"http://www.spotify.com"];
    tag.enabled = @NO;
    return tag;
}

+ (AHTag *)device8 {
    AHTag *tag = [AHTag new];
    tag.category = @"Device";
    tag.title = @"Android Wear";
    tag.color = [UIColor colorWithRed:0.56 green:0.713 blue:1 alpha:1];
    tag.url = [NSURL URLWithString:@"http://www.spotify.com"];
    tag.enabled = @NO;
    return tag;
}

+ (AHTag *)device9 {
    AHTag *tag = [AHTag new];
    tag.category = @"Device";
    tag.title = @"Web Player";
    tag.color = [UIColor colorWithRed:0.56 green:0.713 blue:1 alpha:1];
    tag.url = [NSURL URLWithString:@"http://www.spotify.com"];
    tag.enabled = @YES;
    return tag;
}

#pragma mark - App store

+ (AHTag *)app1 {
    AHTag *tag = [AHTag new];
    tag.category = @"App Store";
    tag.title = @"Books";
    tag.color = [UIColor colorWithRed:0.396 green:0.803 blue:0.64 alpha:1];
    tag.url = [NSURL URLWithString:@"http://appstore.com"];
    tag.enabled = @YES;
    return tag;
}

+ (AHTag *)app2 {
    AHTag *tag = [AHTag new];
    tag.category = @"App Store";
    tag.title = @"Business";
    tag.color = [UIColor colorWithRed:0.396 green:0.803 blue:0.64 alpha:1];
    tag.url = [NSURL URLWithString:@"http://appstore.com"];
    tag.enabled = @YES;
    return tag;
}

+ (AHTag *)app3 {
    AHTag *tag = [AHTag new];
    tag.category = @"App Store";
    tag.title = @"Catalogues";
    tag.color = [UIColor colorWithRed:0.396 green:0.803 blue:0.64 alpha:1];
    tag.url = [NSURL URLWithString:@"http://appstore.com"];
    tag.enabled = @YES;
    return tag;
}

+ (AHTag *)app4 {
    AHTag *tag = [AHTag new];
    tag.category = @"App Store";
    tag.title = @"Education";
    tag.color = [UIColor colorWithRed:0.396 green:0.803 blue:0.64 alpha:1];
    tag.url = [NSURL URLWithString:@"http://appstore.com"];
    tag.enabled = @YES;
    return tag;
}

+ (AHTag *)app5 {
    AHTag *tag = [AHTag new];
    tag.category = @"App Store";
    tag.title = @"Finance";
    tag.color = [UIColor colorWithRed:0.396 green:0.803 blue:0.64 alpha:1];
    tag.url = [NSURL URLWithString:@"http://appstore.com"];
    tag.enabled = @YES;
    return tag;
}

+ (AHTag *)app6 {
    AHTag *tag = [AHTag new];
    tag.category = @"App Store";
    tag.title = @"Food & Drink";
    tag.color = [UIColor colorWithRed:0.396 green:0.803 blue:0.64 alpha:1];
    tag.url = [NSURL URLWithString:@"http://appstore.com"];
    tag.enabled = @YES;
    return tag;
}

+ (AHTag *)app7 {
    AHTag *tag = [AHTag new];
    tag.category = @"App Store";
    tag.title = @"Games";
    tag.color = [UIColor colorWithRed:0.396 green:0.803 blue:0.64 alpha:1];
    tag.url = [NSURL URLWithString:@"http://appstore.com"];
    tag.enabled = @NO;
    return tag;
}

+ (AHTag *)app8 {
    AHTag *tag = [AHTag new];
    tag.category = @"App Store";
    tag.title = @"Health & Fitness";
    tag.color = [UIColor colorWithRed:0.396 green:0.803 blue:0.64 alpha:1];
    tag.url = [NSURL URLWithString:@"http://appstore.com"];
    tag.enabled = @YES;
    return tag;
}

+ (AHTag *)app9 {
    AHTag *tag = [AHTag new];
    tag.category = @"App Store";
    tag.title = @"Kids";
    tag.color = [UIColor colorWithRed:0.396 green:0.803 blue:0.64 alpha:1];
    tag.url = [NSURL URLWithString:@"http://appstore.com"];
    tag.enabled = @YES;
    return tag;
}

+ (AHTag *)app10 {
    AHTag *tag = [AHTag new];
    tag.category = @"App Store";
    tag.title = @"Lifestyle";
    tag.color = [UIColor colorWithRed:0.396 green:0.803 blue:0.64 alpha:1];
    tag.url = [NSURL URLWithString:@"http://appstore.com"];
    tag.enabled = @YES;
    return tag;
}

+ (AHTag *)app11 {
    AHTag *tag = [AHTag new];
    tag.category = @"App Store";
    tag.title = @"Medical";
    tag.color = [UIColor colorWithRed:0.396 green:0.803 blue:0.64 alpha:1];
    tag.url = [NSURL URLWithString:@"http://appstore.com"];
    tag.enabled = @YES;
    return tag;
}

+ (AHTag *)app12 {
    AHTag *tag = [AHTag new];
    tag.category = @"App Store";
    tag.title = @"Music";
    tag.color = [UIColor colorWithRed:0.396 green:0.803 blue:0.64 alpha:1];
    tag.url = [NSURL URLWithString:@"http://appstore.com"];
    tag.enabled = @YES;
    return tag;
}

+ (AHTag *)app13 {
    AHTag *tag = [AHTag new];
    tag.category = @"App Store";
    tag.title = @"Navigation";
    tag.color = [UIColor colorWithRed:0.396 green:0.803 blue:0.64 alpha:1];
    tag.url = [NSURL URLWithString:@"http://appstore.com"];
    tag.enabled = @YES;
    return tag;
}

+ (AHTag *)app14 {
    AHTag *tag = [AHTag new];
    tag.category = @"App Store";
    tag.title = @"News";
    tag.color = [UIColor colorWithRed:0.396 green:0.803 blue:0.64 alpha:1];
    tag.url = [NSURL URLWithString:@"http://appstore.com"];
    tag.enabled = @NO;
    return tag;
}

+ (AHTag *)app15 {
    AHTag *tag = [AHTag new];
    tag.category = @"App Store";
    tag.title = @"Magazines & Newspapers";
    tag.color = [UIColor colorWithRed:0.396 green:0.803 blue:0.64 alpha:1];
    tag.url = [NSURL URLWithString:@"http://appstore.com"];
    tag.enabled = @NO;
    return tag;
}

+ (AHTag *)app16 {
    AHTag *tag = [AHTag new];
    tag.category = @"App Store";
    tag.title = @"Photo & Video";
    tag.color = [UIColor colorWithRed:0.396 green:0.803 blue:0.64 alpha:1];
    tag.url = [NSURL URLWithString:@"http://appstore.com"];
    tag.enabled = @YES;
    return tag;
}

+ (AHTag *)app17 {
    AHTag *tag = [AHTag new];
    tag.category = @"App Store";
    tag.title = @"Productivity";
    tag.color = [UIColor colorWithRed:0.396 green:0.803 blue:0.64 alpha:1];
    tag.url = [NSURL URLWithString:@"http://appstore.com"];
    tag.enabled = @NO;
    return tag;
}

+ (AHTag *)app18 {
    AHTag *tag = [AHTag new];
    tag.category = @"App Store";
    tag.title = @"Reference";
    tag.color = [UIColor colorWithRed:0.396 green:0.803 blue:0.64 alpha:1];
    tag.url = [NSURL URLWithString:@"http://appstore.com"];
    tag.enabled = @YES;
    return tag;
}

+ (AHTag *)app19 {
    AHTag *tag = [AHTag new];
    tag.category = @"App Store";
    tag.title = @"Shopping";
    tag.color = [UIColor colorWithRed:0.396 green:0.803 blue:0.64 alpha:1];
    tag.url = [NSURL URLWithString:@"http://appstore.com"];
    tag.enabled = @YES;
    return tag;
}

+ (AHTag *)app20 {
    AHTag *tag = [AHTag new];
    tag.category = @"App Store";
    tag.title = @"Social Networking";
    tag.color = [UIColor colorWithRed:0.396 green:0.803 blue:0.64 alpha:1];
    tag.url = [NSURL URLWithString:@"http://appstore.com"];
    tag.enabled = @YES;
    return tag;
}

+ (AHTag *)app21 {
    AHTag *tag = [AHTag new];
    tag.category = @"App Store";
    tag.title = @"Sports";
    tag.color = [UIColor colorWithRed:0.396 green:0.803 blue:0.64 alpha:1];
    tag.url = [NSURL URLWithString:@"http://appstore.com"];
    tag.enabled = @YES;
    return tag;
}

+ (AHTag *)app22 {
    AHTag *tag = [AHTag new];
    tag.category = @"App Store";
    tag.title = @"Travel";
    tag.color = [UIColor colorWithRed:0.396 green:0.803 blue:0.64 alpha:1];
    tag.url = [NSURL URLWithString:@"http://appstore.com"];
    tag.enabled = @YES;
    return tag;
}

+ (AHTag *)app23 {
    AHTag *tag = [AHTag new];
    tag.category = @"App Store";
    tag.title = @"Utilities";
    tag.color = [UIColor colorWithRed:0.396 green:0.803 blue:0.64 alpha:1];
    tag.url = [NSURL URLWithString:@"http://appstore.com"];
    tag.enabled = @YES;
    return tag;
}

+ (AHTag *)app24 {
    AHTag *tag = [AHTag new];
    tag.category = @"App Store";
    tag.title = @"Weather";
    tag.color = [UIColor colorWithRed:0.396 green:0.803 blue:0.64 alpha:1];
    tag.url = [NSURL URLWithString:@"http://appstore.com"];
    tag.enabled = @YES;
    return tag;
}

@end

File: Swift/AutomaticHeightTagTableViewCell/Models/TagGroups.swift
//
//  TagGroups.swift
//  AutomaticHeightTagTableViewCell
//
//  Created by WEI-JEN TU on 2016-09-23.
//  Copyright © 2016 Cold Yam. All rights reserved.
//

import UIKit

let pinterest = [pin1, pin2, pin3, pin4]
let genre = [genre1, genre2, genre3, genre4, genre5, genre6, genre7, genre8, genre9, genre10]
let device = [device1, device2, device3, device4, device5, device6, device7, device8, device9]
let app = [app1, app2, app3, app4, app5, app6, app7, app8, app9, app10, app11, app12, app13, app14, app15, app16, app17, app18, app19, app20, app22, app22, app23, app24]

// MARK: - Pinterest

private let pin1 = AHTag(category: "Pinterest", title: "All Pins", color: UIColor(red: 1, green: 0.56, blue: 0.56, alpha: 1), URL: NSURL(string: "http://pinterest.com"), enabled: true)
private let pin2 = AHTag(category: "Pinterest", title: "Your Pins", color: UIColor(red: 1, green: 0.56, blue: 0.56, alpha: 1), URL: NSURL(string: "http://pinterest.com"), enabled: true)
private let pin3 = AHTag(category: "Pinterest", title: "People", color: UIColor(red: 1, green: 0.56, blue: 0.56, alpha: 1), URL: NSURL(string: "http://pinterest.com"), enabled: false)
private let pin4 = AHTag(category: "Pinterest", title: "Boards", color: UIColor(red: 1, green: 0.56, blue: 0.56, alpha: 1), URL: NSURL(string: "http://pinterest.com"), enabled: true)

// MARK: - Genre

private let genre1 = AHTag(category: "Genre", title: "Classical", color: UIColor(red: 0.626, green: 0.56, blue: 1, alpha: 1), URL: NSURL(string: "http://itunes.com"), enabled: true)
private let genre2 = AHTag(category: "Genre", title: "Comedy", color: UIColor(red: 0.626, green: 0.56, blue: 1, alpha: 1), URL: NSURL(string: "http://itunes.com"), enabled: true)
private let genre3 = AHTag(category: "Genre", title: "Country", color: UIColor(red: 0.626, green: 0.56, blue: 1, alpha: 1), URL: NSURL(string: "http://itunes.com"), enabled: false)
private let genre4 = AHTag(category: "Genre", title: "Dance", color: UIColor(red: 0.626, green: 0.56, blue: 1, alpha: 1), URL: NSURL(string: "http://itunes.com"), enabled: false)
private let genre5 = AHTag(category: "Genre", title: "Drama", color: UIColor(red: 0.626, green: 0.56, blue: 1, alpha: 1), URL: NSURL(string: "http://itunes.com"), enabled: true)
private let genre6 = AHTag(category: "Genre", title: "Early Listening", color: UIColor(red: 0.626, green: 0.56, blue: 1, alpha: 1), URL: NSURL(string: "http://itunes.com"), enabled: true)
private let genre7 = AHTag(category: "Genre", title: "Electronic", color: UIColor(red: 0.626, green: 0.56, blue: 1, alpha: 1), URL: NSURL(string: "http://itunes.com"), enabled: false)
private let genre8 = AHTag(category: "Genre", title: "Folk", color: UIColor(red: 0.626, green: 0.56, blue: 1, alpha: 1), URL: NSURL(string: "http://itunes.com"), enabled: false)
private let genre9 = AHTag(category: "Genre", title: "Holiday", color: UIColor(red: 0.626, green: 0.56, blue: 1, alpha: 1), URL: NSURL(string: "http://itunes.com"), enabled: false)
private let genre10 = AHTag(category: "Genre", title: "Mobile", color: UIColor(red: 0.626, green: 0.56, blue: 1, alpha: 1), URL: NSURL(string: "http://itunes.com"), enabled: true)

// MARK: - Device

private let device1 = AHTag(category: "Device", title: "Mobile", color: UIColor(red: 0.56, green: 0.713, blue: 1, alpha: 1), URL: NSURL(string: "http://www.spotify.com"), enabled: true)
private let device2 = AHTag(category: "Device", title: "Computer", color: UIColor(red: 0.56, green: 0.713, blue: 1, alpha: 1), URL: NSURL(string: "http://www.spotify.com"), enabled: true)
private let device3 = AHTag(category: "Device", title: "Tablet", color: UIColor(red: 0.56, green: 0.713, blue: 1, alpha: 1), URL: NSURL(string: "http://www.spotify.com"), enabled: true)
private let device4 = AHTag(category: "Device", title: "Car", color: UIColor(red: 0.56, green: 0.713, blue: 1, alpha: 1), URL: NSURL(string: "http://www.spotify.com"), enabled: false)
private let device5 = AHTag(category: "Device", title: "Speaker", color: UIColor(red: 0.56, green: 0.713, blue: 1, alpha: 1), URL: NSURL(string: "http://www.spotify.com"), enabled: true)
private let device6 = AHTag(category: "Device", title: "Playstation", color: UIColor(red: 0.56, green: 0.713, blue: 1, alpha: 1), URL: NSURL(string: "http://www.spotify.com"), enabled: true)
private let device7 = AHTag(category: "Device", title: "TV", color: UIColor(red: 0.56, green: 0.713, blue: 1, alpha: 1), URL: NSURL(string: "http://www.spotify.com"), enabled: false)
private let device8 = AHTag(category: "Device", title: "Android Wear", color: UIColor(red: 0.56, green: 0.713, blue: 1, alpha: 1), URL: NSURL(string: "http://www.spotify.com"), enabled: false)
private let device9 = AHTag(category: "Device", title: "Web Player", color: UIColor(red: 0.56, green: 0.713, blue: 1, alpha: 1), URL: NSURL(string: "http://www.spotify.com"), enabled: true)

// MARK: - App store

private let app1 = AHTag(category: "App Store", title: "Books", color: UIColor(red: 0.396, green: 0.803, blue:0.64, alpha: 1), URL: NSURL(string: "http://appstore.com"), enabled: true)
private let app2 = AHTag(category: "App Store", title: "Business", color: UIColor(red: 0.396, green: 0.803, blue:0.64, alpha: 1), URL: NSURL(string: "http://appstore.com"), enabled: true)
private let app3 = AHTag(category: "App Store", title: "Catalogues", color: UIColor(red: 0.396, green: 0.803, blue:0.64, alpha: 1), URL: NSURL(string: "http://appstore.com"), enabled: true)
private let app4 = AHTag(category: "App Store", title: "Education", color: UIColor(red: 0.396, green: 0.803, blue:0.64, alpha: 1), URL: NSURL(string: "http://appstore.com"), enabled: false)
private let app5 = AHTag(category: "App Store", title: "Finance", color: UIColor(red: 0.396, green: 0.803, blue:0.64, alpha: 1), URL: NSURL(string: "http://appstore.com"), enabled: true)
private let app6 = AHTag(category: "App Store", title: "Food & Drink", color: UIColor(red: 0.396, green: 0.803, blue:0.64, alpha: 1), URL: NSURL(string: "http://appstore.com"), enabled: true)
private let app7 = AHTag(category: "App Store", title: "Games", color: UIColor(red: 0.396, green: 0.803, blue:0.64, alpha: 1), URL: NSURL(string: "http://appstore.com"), enabled: false)
private let app8 = AHTag(category: "App Store", title: "Health & Fitness", color: UIColor(red: 0.396, green: 0.803, blue:0.64, alpha: 1), URL: NSURL(string: "http://appstore.com"), enabled: false)
private let app9 = AHTag(category: "App Store", title: "Kids", color: UIColor(red: 0.396, green: 0.803, blue:0.64, alpha: 1), URL: NSURL(string: "http://appstore.com"), enabled: true)
private let app10 = AHTag(category: "App Store", title: "Lifestyle", color: UIColor(red: 0.396, green: 0.803, blue:0.64, alpha: 1), URL: NSURL(string: "http://appstore.com"), enabled: true)
private let app11 = AHTag(category: "App Store", title: "Medical", color: UIColor(red: 0.396, green: 0.803, blue:0.64, alpha: 1), URL: NSURL(string: "http://appstore.com"), enabled: true)
private let app12 = AHTag(category: "App Store", title: "Music", color: UIColor(red: 0.396, green: 0.803, blue:0.64, alpha: 1), URL: NSURL(string: "http://appstore.com"), enabled: true)
private let app13 = AHTag(category: "App Store", title: "Navigation", color: UIColor(red: 0.396, green: 0.803, blue:0.64, alpha: 1), URL: NSURL(string: "http://appstore.com"), enabled: false)
private let app14 = AHTag(category: "App Store", title: "News", color: UIColor(red: 0.396, green: 0.803, blue:0.64, alpha: 1), URL: NSURL(string: "http://appstore.com"), enabled: true)
private let app15 = AHTag(category: "App Store", title: "Magazines & Newspapers", color: UIColor(red: 0.396, green: 0.803, blue:0.64, alpha: 1), URL: NSURL(string: "http://appstore.com"), enabled: true)
private let app16 = AHTag(category: "App Store", title: "Photo & Video", color: UIColor(red: 0.396, green: 0.803, blue:0.64, alpha: 1), URL: NSURL(string: "http://appstore.com"), enabled: false)
private let app17 = AHTag(category: "App Store", title: "Productivity", color: UIColor(red: 0.396, green: 0.803, blue:0.64, alpha: 1), URL: NSURL(string: "http://appstore.com"), enabled: false)
private let app18 = AHTag(category: "App Store", title: "Reference", color: UIColor(red: 0.396, green: 0.803, blue:0.64, alpha: 1), URL: NSURL(string: "http://appstore.com"), enabled: true)
private let app19 = AHTag(category: "App Store", title: "Shopping", color: UIColor(red: 0.396, green: 0.803, blue:0.64, alpha: 1), URL: NSURL(string: "http://appstore.com"), enabled: true)
private let app20 = AHTag(category: "App Store", title: "Social Networking", color: UIColor(red: 0.396, green: 0.803, blue:0.64, alpha: 1), URL: NSURL(string: "http://appstore.com"), enabled: true)
private let app21 = AHTag(category: "App Store", title: "Sports", color: UIColor(red: 0.396, green: 0.803, blue:0.64, alpha: 1), URL: NSURL(string: "http://appstore.com"), enabled: true)
private let app22 = AHTag(category: "App Store", title: "Travel", color: UIColor(red: 0.396, green: 0.803, blue:0.64, alpha: 1), URL: NSURL(string: "http://appstore.com"), enabled: false)
private let app23 = AHTag(category: "App Store", title: "Utilities", color: UIColor(red: 0.396, green: 0.803, blue:0.64, alpha: 1), URL: NSURL(string: "http://appstore.com"), enabled: true)
private let app24 = AHTag(category: "App Store", title: "Weather", color: UIColor(red: 0.396, green: 0.803, blue:0.64, alpha: 1), URL: NSURL(string: "http://appstore.com"), enabled: true)
